README.md

```diff
@@ -37,10 +37,7 @@
 
 # 📖 Math Magicians <a name="about-project"></a>
 
```

Comment:
Kindly make sure the title corresponds to your actual project (Task Manager)!

Reply:
Silly me! 😮, you are right @HFG43! thank you to point that out Hernan! I'll make the changes right away 🎯